### DISCUSSION TITLE
How to disable Download Video on ft.Video on Webbrowser both desktop and mobile

### DISCUSSION THREAD

**Discussion by freevid32** [flet-dev/flet#4131] [Q&A]
How to disable Download Video on ft.Video on Webbrowser both desktop and mobile 

<img width="1424" alt="Screenshot 2567-05-15 at 11 20 46" src="https://github.com/flet-dev/flet/assets/141158128/33ecac76-6698-43d1-964e-cd80283ae6cc">

**Comment by ndonkoHenri** [comment]
Try disabling the Browser context menu. [How to?](https://python.plainenglish.io/disable-enable-browser-context-menu-in-a-flet-web-app-python-fcf2d38acaa2)
